pkg/kubelet/lifecycle/predicate_test.go

```diff
@@ -431,3 +435,115 @@ func TestRejectPodAdmissionBasedOnOSField(t *testing.T) {
 		})
 	}
 }
+
+func TestPodAdmissionBasedOnSupplementalGroupsPolicy(t *testing.T) {
+	nodeSupportedTheFeature := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{Name: "test"},
+		Status: v1.NodeStatus{
+			Features: &v1.NodeFeatures{
+				SupplementalGroupsPolicy: ptr.To(true),
+			},
+		},
+	}
+	nodeNotSupportedTheFeature := &v1.Node{
```

Comment:
nit: `nodeWithoutFeature`

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/130210/commits/0ef6d8c70585f295ad869ee3ffddba08295485b3